karma.conf-testingbot.js

```diff
@@ -0,0 +1,57 @@
+var fs = require('fs')
+
+module.exports = function(config) {
+  // Browsers to run on TestingBot
+  var customLaunchers = {
+    TB_Chrome: {
+      base: 'TestingBot',
+      browserName: 'chrome'
+    },
+    TB_Firefox: {
+      base: 'TestingBot',
+      browserName: 'firefox'
+    }
+  }
+
+  config.set({
+    // Base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // Frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+    // List of files / patterns to load in the browser
+    files: ['browserfied/browser.js', 'test/*.js'],
```

Comment:
nit: Not a big fan of the name 'browserified'. It's descriptive, but also is somewhat tied to browserify which is not what is happening here. Naming is hard and we have used dist for the NPM distribution, so I unfortunately do not have an alternate good suggestion at the moment.

Reply:
browser-ready?
vanillaJS?
plainScripts?
compiledScripts?
webpacked?

Reply:
hmm, none of these really feel like they fit for me. I almost wonder if we should adjust and have a `dist` folder that contains `browser` and `npm` subfolderss which we then use for distribution. This matters very little in the grander scheme of things, but I would expect our transpiled things to live in `/dist` as a general rule.